TTHAnalysis/python/plotter/susy-sos/fr-MET.txt

```diff
@@ -0,0 +1,2 @@
+change: met_pt  : met_genPt
```

Comment:
@peruzzim (including @cbotta ): could you please remind me the meaning of \b?
This fr file was used for applying the ICHEP recommendation in the treatment of MET uncertainty in fastsim (--> it is used in the signal mca)